src/build/tenscript/mod.rs

```diff
@@ -27,16 +27,33 @@ mod build_phase;
 pub struct TenscriptParser;
 
 #[derive(Debug)]
-pub enum ParseError {
+pub enum TenscriptError {
     Pest(Error<Rule>),
+    Format(String),
     Invalid(String),
 }
 
-impl Display for ParseError {
+impl TenscriptError {
```

Comment:
Weird to put these convenience methods inside an impl called `TenscriptError`.

Reply:
Suggestion?

Reply:
Weird is not an argument.  What would be not weird?